Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -268,6 +297,17 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
         collectionView.scrollToMostRecentMessage(animated: animated)
     }
     
+    open func updateScrollToLatestMessageButton() {
```

Comment:
nit 1: this `open func` doesn't have documentation which would probably help with nit 2 😝
nit 2:  `updateScrollToLatestMessageButton` sounds like it returns `UIButton` subclass... what about naming it  `setScrollToLatestMessageButton(visible: Bool)`? It's even more flexible when some customer would want to implement custom logic for hiding/showing the button, what do you think?

Reply:
This function contains logic when to show/hide, I would have to implement another function to show and hide the button. And it felt too bloated with functions for such a "simple" task. but now thinking about it, when you want to change the animation or anything, it would be a separated function would be a good place to do it 👍🏻